### ISSUE TITLE
calamares: main logo is skewed in upper corner

### ISSUE THREAD

**Issue by star-buck** [blackbird-os/issues#2]
![Screenshot_20190706_152115](https://user-images.githubusercontent.com/840070/60756764-23ea4080-9ff1-11e9-8736-f7f622ced402.png)

**Comment by NuLogicSystems** [comment]
Yes, I already mentioned this. I'm not sure how to fix this other than use just the icon here.

**Comment by NuLogicSystems** [comment]
That's odd, git-cola closed the issue instead of referencing it.

**Comment by anachronox** [comment]
Here are dimensions for the logo size:
https://github.com/calamares/calamares/blob/3a7d0d4ee7e170ca974266f1a7b5692d43221f5c/src/branding/default/branding.desc#L76

productLogo is 80x80 px. Workaround, without touching the code, is to create a square png image with transparent background and modify the logo to fit in the middle.

![calamares-product-logo](https://user-images.githubusercontent.com/1354530/67152410-8a137e80-f2d5-11e9-8976-c45a37802f6c.png)
